docs/alt-installation.md

```diff
@@ -50,3 +50,21 @@ github.com/earthly/hello-world:main+hello | Hello, world!
 github.com/earthly/hello-world:main+hello | Target github.com/earthly/hello-world:main+hello built successfully
 =========================== SUCCESS ===========================
 ```
+
+# Removal
```

Comment:
Perhaps?

```suggestion
# Uninstall
```

Reply:
good plan, that way searching for either "uninstall" or "remove" will hit this section (since the first line of text include the word remove).